src/include/daos/mgmt.h

```diff
@@ -58,7 +58,8 @@ void dc_mgmt_sys_detach(struct dc_mgmt_sys *sys);
 ssize_t dc_mgmt_sys_encode(struct dc_mgmt_sys *sys, void *buf, size_t cap);
 ssize_t dc_mgmt_sys_decode(void *buf, size_t len, struct dc_mgmt_sys **sysp);
 
-int dc_mgmt_net_cfg(const char *name);
+int
+     dc_mgmt_net_cfg(const char *name, crt_init_options_t *crt_info);
```

Comment:
strange formatting:)

Reply:
that's not me, it's the auto clang format from the githook.